### PR TITLE
Project Creating #1

### PR DESCRIPTION
**Pull request by af150416**
This is a working project

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	compile('org.springframework.boot:spring-boot-starter-data-jpa')
+	compile('org.springframework.boot:spring-boot-starter-jdbc')
+	compile('org.springframework.boot:spring-boot-starter-web')
+	compile group: 'com.zaxxer', name: 'HikariCP', version: '2.5.1'
+	compile group: 'org.springframework', name: 'spring-jdbc', version: '4.3.5.RELEASE'
+	compile group: 'mysql', name: 'mysql-connector-java', version: '6.0.5'
+	compile 'org.mockito:mockito-core:2.5.4'
+	runtime('com.h2database:h2')
+	runtime('mysql:mysql-connector-java')
+}
```
Comment (CortesV) [inline build.gradle]: Add versions of libraries

File: src/main/java/com/github/af150416/plane/controller/IPlane.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+public interface IPlane {
+	/**
+	 * Looking information about plane by id
+	 * 
+	 * @param id
+	 * @return
+	 */
+	public Plane surchByID(int id);
+
```
Comment (Yagi7337) [inline src/main/java/com/github/af150416/plane/controller/IPlane.java]: add access modifiers (public or private or protected)

File: src/main/java/com/github/af150416/plane/controller/IPlane.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	void updatePlane(Plane plane);
```
Comment (Yagi7337) [inline src/main/java/com/github/af150416/plane/controller/IPlane.java]: Add access modifiers (public or private or protected)

File: build.gradle (hunk 1 above)
Comment (af150416) [inline build.gradle]: libraries version was added

File: src/main/java/com/github/af150416/plane/controller/IPlane.java (hunk 3 above)
Comment (af150416) [inline src/main/java/com/github/af150416/plane/controller/IPlane.java]: access modifier was added

**Pull request closed without merging**